### PR TITLE
Spanish translation

### PR DESCRIPTION
**Pull request by ByPaco10**
Spanish (Spain)

### CODE REVIEW

**Comment by ByPaco10** [comment]
Need review from another Spanish speaker and some tweaks to language used

**Comment by Plenttz** [comment]
I approve the translation

**Comment by ByPaco10** [comment]
Translation is clean, ready to merge

**Comment by KingContaria** [comment]
Thank you for translating and for confirming translations, will do the final check later. :)

File: src/main/resources/assets/seedqueue/lang/es_es.json
Comment (KingContaria) [inline src/main/resources/assets/seedqueue/lang/es_es.json:99]: "Sets the thread priority for seeds generating in the background." from the english translation refers to seeds generating in the background, not minecraft in the background. The server thread is the thread for a seed in the queue.

Comment (ByPaco10) [reply src/main/resources/assets/seedqueue/lang/es_es.json]: Fixed on latest commit. Thanks

**Review by KingContaria — APPROVED** [review]
Checked, looks good to me

**Comment by KingContaria** [comment]
With 1.2 there is a couple of new/changed translation keys. I want to release this version fairly soon, if you don't have the time to make the edits i can do the renaming and merge the translations, new translations can be added later.

Renamed:
- "speedrunapi.config.seedqueue.option.backgroundPreviews" -> "speedrunapi.config.seedqueue.option.preparingPreviews"
- "speedrunapi.config.seedqueue.option.backgroundPreviews.description" -> "speedrunapi.config.seedqueue.option.preparingPreviews.description"

Added:
- "speedrunapi.config.seedqueue.option.preparingPreviews.value.-1": "Auto"
- "speedrunapi.config.seedqueue.option.resumeOnFilledQueue": "Resume On Filled Queue"
- "speedrunapi.config.seedqueue.option.resumeOnFilledQueue.description": "Whether to resume generating seeds after the queue is filled.\nThis overrides the Max World Generation % option after the queue capacity is filled."
- "seedqueue.key.scheduleJoin": "Schedule Instance"
- "seedqueue.key.scheduleAll": "Schedule All"

**Comment by KingContaria** [comment]
Sorry for this, one more update with 4 new translations was added. Like I said before, absolutely no pressure, I will still merge the translations without the updates if you dont have time. :)

Added:
- "speedrunapi.config.seedqueue.option.chunkMapFreezing": "Chunkmap Freezing"
- "speedrunapi.config.seedqueue.option.chunkMapFreezing.description": "Freezes the chunkmap after the set milliseconds. Useful for the \"Village Resetting\" strategy. Only applies to worlds on the wall that aren't locked."
- "speedrunapi.config.seedqueue.option.chunkMapFreezing.value": "%sms"
- "speedrunapi.config.seedqueue.option.chunkMapFreezing.value.-1": "OFF"

**Comment by ByPaco10** [comment]
I can't do it today, if you are okay with having it tomorrow. I can do it

**Comment by KingContaria** [comment]
Yea absolutely, I'm planning to release sometime this weekend :)

**Comment by ByPaco10** [comment]
Second Batch done, I'm doing the first one now

**Comment by ByPaco10** [comment]
translating "seedqueue.key.scheduleJoin": "Schedule Instance" and "seedqueue.key.scheduleAll": "Schedule All" is going to be a mayor issue unless we have some sort of desc for it (or someone far smarter than me has a translation), in the meantime, I'm leaving it at default English. Everything else is done.

**Comment by KingContaria** [comment]
> translating "seedqueue.key.scheduleJoin": "Schedule Instance" and "seedqueue.key.scheduleAll": "Schedule All" is going to be a mayor issue unless we have some sort of desc for it (or someone far smarter than me has a translation), in the meantime, I'm leaving it at default English. Everything else is done.

Scheduling an instance means it will be played as soon as it finishes generating, Schedule All schedules all currently locked instances so the first one to finish generating will be played.
It probably makes sense to add descriptions for keybindings in the future

**Pull request merged**
